mobile/src/reducers/organization/organization.epics.ts

```diff
@@ -5,36 +5,50 @@ import { Department } from './department.model';
 import { ajaxGetJSON } from 'rxjs/observable/dom/AjaxObservable';
 import { AppState } from '../app.reducer';
 import { Employee } from './employee.model';
+import { Observable } from 'rxjs/Observable';
+import { Alert } from 'react-native';
 
 const url = 'http://localhost:5000/api'; //TODO: fix hardcode
 
 export const loadDepartmentsEpic$ = (action$: ActionsObservable<LoadDepartments>) =>
     action$.ofType('LOAD-DEPARTMENTS')
         .switchMap(x => ajaxGetJSON(`${url}/departments`))
         .map(x => deserializeArray(x as any, Department))
-        .map(x => loadDepartmentsFinished(x));
+        .map(x => loadDepartmentsFinished(x))
+        .catch(e => { 
+            Alert.alert('Error', 'Server is not available')
```

Comment:
Ok, let's do the following

Reply:
remove Alert.alert from here

Reply:
add an Action interface and action creator for these errors

Reply:
Add a separate epic that listens all such error actions and makes Alert call inside its `map` function. We'll need to check though that it's really called and nothing breaks :)

Reply:
Can you try this code as said 'separate epic'? 
```
export const errorEpic= action$ =>
  action$.ofType(MY_MEGA_ERROR)
    .do(({ payload }) =>
      Alert.alert('Error', `error occurred: ${payload.message}`)
    )
    .ignoreElements();
```